rules_gapic/csharp/csharp_gapic.bzl

```diff
@@ -121,7 +121,7 @@ def csharp_gapic_srcjar(name, src, gapic_yaml, service_yaml, **kwargs):
         **kwargs
     )
 
-    _csharp_gapic_postprocessed_srcjar(
+    return _csharp_gapic_postprocessed_srcjar(
```

Comment:
This return doesn't seem to be added by other languages.

Reply:
Sorry... I was doing experiments and did not cleanup stuff properly (this and the php plugin instead of csharp =)